src/main/java/jdbchomework/console/MenuContents.java

```diff
@@ -3,28 +3,25 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class ListMenu {
-
-    private static List<String> siteHeader = new ArrayList<>();
-    private static List<String> serviceHeader = new ArrayList<>();
-    private static List<String> serviceHeaderCompany = new ArrayList<>();
-    private static List<String> serviceHeaderCustomer = new ArrayList<>();
-    private static List<String> serviceHeaderDeveloper = new ArrayList<>();
-    private static List<String> serviceHeaderProject = new ArrayList<>();
-    private static List<String> serviceHeaderSkill = new ArrayList<>();
-    private static List<String> serviceMainMenu = new ArrayList<>();
-    private static List<String> serviceCompaniesMenu = new ArrayList<>();
-    private static List<String> serviceCustomersMenu = new ArrayList<>();
-    private static List<String> serviceDevelopersMenu = new ArrayList<>();
-    private static List<String> serviceProjectsMenu = new ArrayList<>();
-    private static List<String> serviceSkillsMenu = new ArrayList<>();
-    private static List<String> selectionHeader = new ArrayList<>();
-    private static List<String> selectionMenu = new ArrayList<>();
-
-    private ListMenu() {
-    }
-
-    public static void makeMenus() {
+public class MenuContents {
+
+    private List<String> siteHeader = new ArrayList<>();
+    private List<String> serviceHeader = new ArrayList<>();
+    private List<String> serviceHeaderCompany = new ArrayList<>();
+    private List<String> serviceHeaderCustomer = new ArrayList<>();
+    private List<String> serviceHeaderDeveloper = new ArrayList<>();
+    private List<String> serviceHeaderProject = new ArrayList<>();
+    private List<String> serviceHeaderSkill = new ArrayList<>();
+    private List<String> serviceMainMenu = new ArrayList<>();
+    private List<String> serviceCompaniesMenu = new ArrayList<>();
+    private List<String> serviceCustomersMenu = new ArrayList<>();
+    private List<String> serviceDevelopersMenu = new ArrayList<>();
+    private List<String> serviceProjectsMenu = new ArrayList<>();
+    private List<String> serviceSkillsMenu = new ArrayList<>();
+    private List<String> selectionHeader = new ArrayList<>();
+    private List<String> selectionMenu = new ArrayList<>();
+
+    public void makeMenus() {
         siteHeader.add("**************************************************************");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "**************************************************************" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)